frontend_tests/node_tests/rendered_markdown.js

```diff
@@ -181,7 +181,7 @@ run_test("timestamp", () => {
     rm.update_elements($content);
 
     // Final asserts
-    assert.equal($timestamp.text(), "Thu, Jan 1 1970, 12:00 AM");
+    assert.equal($timestamp.text(), "never-been-set");
```

Comment:
You can't make tests pass by simply changing the expected values.  The `never-been-set` here should be a clue that you need to change your test setup here, or possibly change the implementation to be easier to test.